inference-engine/src/inference_engine/src/cpp/ie_executable_network.cpp

```diff
@@ -133,12 +133,82 @@ std::shared_ptr<const Function> ExecutableNetwork::get_runtime_function() const
     OV_EXEC_NET_CALL_STATEMENT(return std::const_pointer_cast<const Function>(_impl->GetExecGraphInfo()));
 }
 
-ParameterVector ExecutableNetwork::get_parameters() const {
-    OV_EXEC_NET_CALL_STATEMENT(return _impl->GetExecGraphInfo()->get_parameters());
+std::vector<ov::Output<const ov::Node>> ExecutableNetwork::inputs() const {
+    // clang-format off
+    OV_EXEC_NET_CALL_STATEMENT(
```

Comment:
we can use `OV_EXEC_NET_CALL_STATEMENT({ xxx })` and clang will provide good formatting